_projects/quickstart_green_mode.md

```diff
@@ -0,0 +1,96 @@
+---
+collaborating_projects:
+  - coala-quickstart
+  - coala
+desc: "Generate project compatible config files."
+difficulty: medium
+initiatives:
+- GSoC
+- RGSoC
+issues:
+markdown: quickstart_green_mode.md
+mentors:
+  - satwikkansal
+  - jayvdb
+name: "Quickstart Green Mode"
+requirements:
+- "The applicant should have at least one patch accepted to coala-quickstart
+  repository."
+- "The participant should have an understanding of coala's structure and
+  config files."
+- "The proposal needs to contain a clear concept."
+tags:
+- coala-quickstart
+- coala-ci
+---
+
+coala doesn't help the user unless the user is familiar with its settings and
+bears. coala-quickstart is the project to handle this situation by asking the
+users general questions about their project and producing a config file to lint
+their code.
+
+Currently coala-quickstart asks the user a lot of questions, and usually
```

Comment:
fwiw, coala quickstart also has a [`--ci` mode](https://github.com/coala/coala-quickstart/blob/master/coala_quickstart/coala_quickstart.py#L44) (I'm not sure how exactly it works), where it doesn't ask any questions. So I guess this project would involve enhancing the `--ci` mode.

Reply:
I think we can mention this fact in the project description.

Reply:
`--green` would be in addition to `--ci`.

`--ci` skips questions that typically require a user.

`--green` determines optimal answers by running `coala` and learning from its output.

Reply:
I see, makes sense.

Reply:
description regarding this added